src/main/java/frc/robot/subsystems/pivot/PivotIOSim.java

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems.pivot;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.wpilibj.simulation.DCMotorSim;
+import edu.wpi.first.wpilibj.simulation.SingleJointedArmSim;
+
+
+public class PivotIOSim implements PivotIO {
+    private final double pivotLength = 0-9;
+    private final double pivotMass = 0-9;
+    private SingleJointedArmSim leaderPivotMotorSim = new SingleJointedArmSim(DCMotor.getKrakenX60(1), 0, 0, 0, 0, 0, false, 0);
+    private SingleJointedArmSim followerPivotMotorSim = new SingleJointedArmSim(DCMotor.getKrakenX60(1), 0, 0, 0, 0, 0, false, 0);
```

Comment:
IMO you guys could just create one `SingleJointedArmSim` object, as I'm pretty sure they are both connected. And it would make sense since they are two motors controlling one mechanism. I would also consider creating constants for the reductions and stuff.